docs/docsite/collection-plugins.yml

```diff
@@ -0,0 +1,17 @@
+# We also need an example of modules hosted in Automation Hub
```

Comment:
I don't know if docs.ansible.com should ever point to docs on AH module/plugin docs.

Reply:
<nid> yeah, if we only point at things in Galaxy, c then we won't need an automation hub hosted example

Reply:
agreed: docs.ansible.com should point to Galaxy only